tsconfig.json

```diff
@@ -6,13 +6,12 @@
     "moduleResolution": "node",
     "strict": true,
     "isolatedModules": true,
-    "useDefineForClassFields": false,
     "jsx": "preserve",
     "resolveJsonModule": true,
     "esModuleInterop": true,
     "sourceMap": true,
     "skipLibCheck": true,
-    "experimentalDecorators": true,
+    "noUnusedLocals": true,
```

Comment:
Since the current eslint config skips detection of unused vars, this fallback prevents projects inheriting rogue types/assignments.

Reply:
Ah, maybe we should bring back eslint rule...? I think it was turned off during the refactoring, I'm not sure. Can you check what errors do we get by turning back eslint rule on?

Reply:
Removed. It has no impact, but would have picked up some of the unused imports I've removed in this PR. 👍

Reply:
FYI @globalbrain/eslint-config ensures no mistakes are made. But that's another topic and PR. 👀

Reply:
Nice!